packages/devops/scripts/ci/deployBackend.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 PACKAGE=$1
-DIR=`dirname "$0"`
+DIR=$(dirname "$0")
```

Comment:
`shell-format` prefers `$()` over ``. It's probably fine either way, but being consistent is probably beneficial.